src/main/java/frc/robot/util/SafetyChecker.java

```diff
@@ -24,19 +24,19 @@ public static boolean isSafe(
 
   // Specific safety implementations
   private static boolean checkArmElevatorSafety(double elevatorPosition, double armPosition) {
-    return elevatorPosition > Constants.ElevatorConstantsLeonidas.ARM_FACTORY_MIN_POS
+    return elevatorPosition > Constants.ArmConstantsLeonidas.ARM_OPERATIONAL_MIN_POS
```

Comment:
Why are we comparing an elevatorPosition to an ARM constant?  Please fix - this is important as it could render the safety checker invalid.